src/js/media-chrome-button.ts

```diff
@@ -292,10 +294,9 @@ class MediaChromeButton extends globalThis.HTMLElement {
     this.#mediaController?.unassociateElement?.(this);
     this.#mediaController = null;
 
-    this.removeEventListener('mouseenter', this.tooltipEl?.updateXOffset);
-    this.removeEventListener('focus', this.tooltipEl?.updateXOffset);
+    this.removeEventListener('mouseenter', this.#positionTooltip);
+    this.removeEventListener('focus', this.#positionTooltip);
     this.removeEventListener('click', this.#clickListener);
-    this.tooltipEl = null;
```

Comment:
on disconnected we don't want to empty this property because we only query it in the constructor, not in connectedCallback